### PR TITLE
Fix UIComponent.js's IsVisible method, and add a test in geometry/ui/Geometry.InfowindowS…

### PR DESCRIPTION
**Pull request by liubgithub**
It can not judge whether the infowindow is visible or not only by its style 'display!="none"'

### CODE REVIEW

File: src/ui/UIComponent.js [hunk 1]
```diff
@@ -273,7 +273,7 @@ class UIComponent extends Eventable(Class) {
      * @returns {Boolean} true|false
      */
     isVisible() {
-        return this.getMap() && this.getDOM() && this.getDOM().style.display !== 'none';
+        return this.getMap() && this.getDOM() && this.__uiDOM.parentNode !== null && this.getDOM().style.display !== 'none';
```
Comment (fuzhenn) [inline src/ui/UIComponent.js]: please change to:
```js
const dom = this.getDOM();
return this.getMap() && dom && dom.parentNode && dom.style.display !== 'none';
```

File: test/geometry/ui/Geometry.InfowindowSpec.js [hunk 2]
```diff
@@ -474,6 +474,32 @@ describe('Geometry.InfoWindow', function () {
...
+            layer.addGeometry(geo, geo2);
+            
+            infoWindow1.addTo(geo).show();
+            infoWindow2.addTo(geo2);
+
+            setTimeout(function () {
+               infoWindow2.show();
+               expect(infoWindow1.isVisible()).not.to.be.ok();
+               done();
+            }, 50);
```
Comment (fuzhenn) [inline test/geometry/ui/Geometry.InfowindowSpec.js]: change to 80

Comment (liubgithub) [reply test/geometry/ui/Geometry.InfowindowSpec.js]: why should change to 80

Comment (fuzhenn) [reply test/geometry/ui/Geometry.InfowindowSpec.js]: 50 is too short for some browsers to execute in such a time window.
80 is much more safer.

**Pull request merged**